hs_core/templates/pages/genericresource.html

```diff
@@ -118,12 +116,17 @@
     ],
     {% endif %}
     {% if fundingagencies %}
-    "funder": [
+    "funding": [
     {% for agency in fundingagencies %}
         {
-            "@type": "Organization",
-            "name": "{{ agency.agency_name | escapejs }}",
-            "url": "{{ agency.agency_url }}"
+            "@type": "Grant",
+            "identifier": "{{ agency.award_number }}",
+            "name": "{{ agency.award_title }}",
```

Comment:
```suggestion
            "name": "{{ agency.award_title | escapejs }}",
```

Reply:
@devincowan Good catch. Have just added the escapejs filter. See if it looks good now.